core/services/eth_request_events.go

```diff
@@ -0,0 +1,94 @@
+package services
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink/core/services/job"
+	"github.com/smartcontractkit/chainlink/core/services/pipeline"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"gopkg.in/guregu/null.v4"
+)
+
+// EthRequestEvent is a wrapper for `models.EthRequestEvent`, the DB
+// representation of the job spec. It fulfills the job.Spec interface
+// and has facilities for unmarshaling the pipeline DAG from the job spec text.
+type EthRequestEventSpec struct {
+	// TODO: Abstract these fields?
```

Comment:
Ticket or no TODO 🙂 I also don't particularly like this given we have job spec v2 and a lot of (all?) of these fields are duplicated here, but we already had this conversation 🤷‍♂️

Reply:
Well, I suppose this is a request to any reviewers to weigh in with their thoughts